README.md

```diff
@@ -1,32 +1,44 @@
-Handles End of Day (EoD) slash commands and sending EoD reminders.
-
-## Usage
-
-- `/eod`: submits an EoD
-- `/eods_left`: returns a list of who have not submitted their EoD for the day
-- `/set_eod_reminder_channel`: sets the EoD reminder channel
-- `/add_eod_reminder`: adds an EoD reminder
-- `/check_eod_reminders`: returns a list of your EoD reminders
-- `/remove_eod_reminder`: remove an EoD reminder
-
-## Initial setup
-
-Note: installation scripts only support Linux distributions at the moment
-
-1. Install systemd package if you don't have it already
-
-2. Install npm packages
-
-   `npm install`
-
-3. Run the installation script for Osteppy services
-
-   `Osteppy/src/osteppy_installation.sh`
-
-## Instructions
-
-### Set up slack token
-
-Store the Slack token as plain text in `Osteppy/config-files/SLACK_TOKEN`
-
-Note: This key is called from the backend, so it should not be available to the public.
+Server backend for Osteppy App - OSTEP that:
+
+- Manage EoDs (End of Day).
+- Check unintended open ports on research machines.
+
+## Slash commands on Slack
+
+- `/eod`: submits an EoD. EoD can be re-submitted within one day, it'll replace the previous submitted EoD.
+- `/eod :<icon>:` same as above but with custom Slack icon.
```

Comment:
Is this how the variant with icon is used?

Reply:
Yes, I think it has always been like that.